liquibase-core/pom.xml

```diff
@@ -336,7 +322,7 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Looks like when you replaced it with gmaven, this maven-compiler-plugin became redudant